webpack.conf.js

```diff
@@ -1,28 +1,68 @@
-var webpack = require('webpack');
+var CleanWebpackPlugin = require('clean-webpack-plugin'),
+    config = require('./config.json'),
+    LodashModuleReplacementPlugin = require('lodash-webpack-plugin'),
+    path = require('path'),
+    webpack = require('webpack');
 
 module.exports = {
+    watch: false,
     devtool: 'source-map',
+    context: __dirname,
+    entry: {
+        main: './assets/js/app.js',
+    },
+    output: {
+        filename: 'theme-bundle.[name].js',
+        path: path.resolve(__dirname, "assets/dist")
+    },
     bail: true,
     module: {
-        loaders: [
+        rules: [
             {
                 test: /\.js$/,
-                loader: 'babel',
                 include: /(assets\/js|assets\\js|stencil-utils)/,
-                query: {
-                    compact: false,
-                    cacheDirectory: true,
-                    presets: ['es2015-loose'],
+                use: {
+                    loader: 'babel-loader',
+                    options: {
+                        compact: false,
+                        cacheDirectory: true,
+                        presets: [
+                            ['latest', {
```

Comment:
Have you looked at using [babel-preset-env](https://github.com/babel/babel-preset-env) instead of using latest? It ties in really nicely with the [autoprefixer](https://github.com/bigcommerce/cornerstone/blob/master/config.json#L37) list already used in Stencil. It's the same basic functionality as `latest`, but lets you keep some of the newer stuff provided your target browsers are fairly modern.

```js
var webpack = require('webpack');
var config = require('./config');

module.exports = {
    devtool: 'source-map',
    bail: true,
    module: {
        loaders: [
            {
                test: /\.js$/,
                loader: 'babel-loader',
                include: /(assets\/js|assets\\js|node_modules\/@bigcommerce\/stencil-utils|node_modules\\@bigcommerce\\stencil-utils)/,
                query: {
                    compact: false,
                    cacheDirectory: true,
                    presets: [
                        ['env', {
                            'targets': {
                                'browsers': config['autoprefixer_browsers'],
                                'uglify': true
                            }
                        }],
                        'stage-3'
                    ],
                    plugins: ['transform-react-jsx']
                }
            }
        ]
    },
    plugins: [
        new webpack.ProvidePlugin({
            $: 'jquery',
            jQuery: 'jquery',
            'window.jQuery': 'jquery'
        })
    ],
    watch: false
};
```